src/fs.zig

```diff
@@ -752,7 +752,7 @@ pub const FileSystem = struct {
         pub fn needToCloseFiles(rfs: *const RealFS) bool {
             // On Windows, we must always close open file handles
             // Windows locks files
-            if (comptime !FeatureFlags.store_file_descriptors) {
+            if (!FeatureFlags.store_file_descriptors or Environment.isWindows) {
```

Comment:
Should we make this `FeatureFlag.store_file_descriptors` defined with`Environment.isWindows` in mind so if referenced in other places it will already be accounted for?

Reply:
jarred decided

i should revert this to how it was before
and actually implement it. the comment is old and incorrect.